project-fob/Controllers/HomeController.cs

```diff
@@ -132,9 +128,17 @@ public ActionResult MeetingPageUser(string meetingId, string password)
             return View("Index");
         }
 
+        public static string GenerateGuid()
+        {
+            Guid g;
+            g = Guid.NewGuid();
+
+            return g.ToString("N");
```

Comment:
This could be just  `return Guid.NewGuid().ToString("N");` Also, it looks like this might not need to be public because it's only used in the `GenerateId()` method.